screens/Home.js

```diff
@@ -221,60 +221,62 @@ class Home extends React.Component {
     return (
       <SafeAreaView
         style={{
-          flex: 1, 
-          backgroundColor: GLOBAL.COLOR.TOOLBAR_COLOR
-        }}>
-         
+          flex: 1,
+          backgroundColor: "#ecf0f1",
```

Comment:
It should not be hard coded. The way it was written previously should be followed. If there is any change in color you can create new variable